### PR TITLE
Activation support (in progress)

### PR DESCRIPTION
**Pull request by asilvas**
Some of the activations weren't trivial due to missing functions such as `tileAs`, `clamp`, `erf` (lacks gradient fn).

### CODE REVIEW

**Comment by bwasti** [comment]
oh wow, I didn't catch this before pushing up relu/leakyRelu just now! (sorry about that, didn't mean to trample)

these look great, but I'm curious why you've chosen a module based approach?  there aren't any parameters so I think we can go with a functional version

if you're interested, here are some ideas to think about including in the PR:
- swish (https://en.wikipedia.org/wiki/Swish_function)
- relu6 (https://pytorch.org/docs/stable/generated/torch.nn.ReLU6.html)
- hardswish (https://pytorch.org/docs/stable/generated/torch.nn.Hardswish.html)
- gelu (https://pytorch.org/docs/stable/generated/torch.nn.GELU.html#torch.nn.GELU)
- elu (https://pytorch.org/docs/stable/generated/torch.nn.ELU.html)

**Comment by asilvas** [comment]
I had modeled it off [Flashlight activations](https://github.com/flashlight/flashlight/blob/5b4441994e7742ac08793204cccd74a73e9079c3/flashlight/fl/nn/modules/Activations.cpp) but have no issue exposing as just another op, so I'll go that route.

**Comment by bwasti** [comment]
looks great! thanks for adding these :)
